frontends/mit-open/src/pages/HomePage/BrowseTopicsSection.tsx

```diff
@@ -119,23 +100,20 @@ const BrowseTopicsSection: React.FC = () => {
       <Container>
         <Title variant="h2">Browse by Topics</Title>
         <Topics>
-          {topics?.results.map(
-            ({ id, name, channel_url: channelUrl }, index) => {
-              const Icon = ICONS[index % ICONS.length]
-              return (
-                <TopicBox key={id} to={channelUrl!}>
-                  <TopicBoxContent>
-                    <Icon />
-                    <TopicBoxName>{name}</TopicBoxName>
-                    <RiArrowRightLine />
-                  </TopicBoxContent>
-                </TopicBox>
-              )
-            },
-          )}
+          {topics?.results.map(({ id, name, channel_url: channelUrl }) => {
+            return (
+              <TopicBox key={id} to={channelUrl!}>
+                <TopicBoxContent>
+                  <RootTopicIcon name={name} />
+                  <TopicBoxName>{name}</TopicBoxName>
+                  <RiArrowRightLine />
+                </TopicBoxContent>
+              </TopicBox>
+            )
+          })}
         </Topics>
-        <SeeAllButton edge="rounded" size="large">
-          See all (coming soon)
+        <SeeAllButton href="/topics/" edge="rounded" size="large">
+          See all
```

Comment:
:tada:

I did notice though that the scroll position is preserved through route navigation, so the Topics Listing page loads half way down when we click this. I'd expect scroll to top to be default behavior, but having had a Google, seems we need to roll our own, e.g. https://dev.to/kunalukey/scroll-to-top-when-route-changes-reactjs-react-router-3bgn. (Also not directly related to this PR)